course/lesson-17-while-loops/lesson.tres

```diff
@@ -0,0 +1,237 @@
+[gd_resource type="Resource" load_steps=23 format=2]
+
+[ext_resource path="res://resources/Lesson.gd" type="Script" id=1]
+[ext_resource path="res://resources/ContentBlock.gd" type="Script" id=2]
+[ext_resource path="res://resources/QuizChoice.gd" type="Script" id=3]
+[ext_resource path="res://course/Documentation.tres" type="Resource" id=4]
+[ext_resource path="res://resources/Practice.gd" type="Script" id=5]
+
+[sub_resource type="Resource" id=1]
+script = ExtResource( 2 )
+content_id = "res://course/lesson-17-while-loops/content-DizKUdOC.tres"
+title = ""
+type = 0
+text = "Our new knowledge of vectors can help move the robot move in a grid.
+
+We can define the size of a board using a [code]Vector2[/code].
+
+Each square is a [i]cell[/i]."
+visual_element_path = ""
+reverse_blocks = false
+has_separator = false
+
+[sub_resource type="Resource" id=2]
+script = ExtResource( 2 )
+content_id = "res://course/lesson-17-while-loops/content-GjB7tKRH.tres"
+title = ""
+type = 0
+text = ""
+visual_element_path = "visuals/ExampleBoard.tscn"
+reverse_blocks = false
+has_separator = false
+
+[sub_resource type="Resource" id=3]
+script = ExtResource( 2 )
+content_id = "res://course/lesson-17-while-loops/content-Q2XNwQMq.tres"
+title = ""
+type = 0
+text = "Let's add our robot onto the grid.
```

Comment:
I would also first show this example of the robot moving on a grid and then break down how the grid works. Basically I feel the start of the lesson is lacking context: the title is about loops, but you talk immediately about a grid and moving on a grid.